fastlane/Fastfile

```diff
@@ -106,6 +106,44 @@ end
       release_version: options[:version])
   end
 
+  # This lane lacks documentation because it ought to be part of the localized
+  # metadata download script instead!
+  desc "Updates the files with the localized keywords values for App Store Connect to match the 100 characters requirement"
+  lane :sanitize_appstore_keywords do | options |
+    Dir["./metadata/**"].each do |locale_dir|
+      keywords_path = File.join(locale_dir, 'keywords.txt')
+
+      unless File.exists?(keywords_path)
+        UI.message "Could not find keywords file in #{locale_dir}. Skipping."
+        next
+      end
+
+      keywords = File.read(keywords_path)
+      app_store_connect_keywords_length_limit = 100
+
+      if keywords.length <= app_store_connect_keywords_length_limit
+        UI.verbose "#{keywords_path} has less than #{app_store_connect_keywords_length_limit} characters. Not trimming."
+        next
+      end
+
+      UI.message "#{keywords_path} has more than #{app_store_connect_keywords_length_limit} characters. Trimming it..."
+
+      case File.basename(locale_dir)
+      when 'ar-SA'
+        separator = '،'
+      else
+        separator = ','
+      end
+
+      until keywords.length <= app_store_connect_keywords_length_limit do
+        keywords = keywords.split(separator)[0...-1].join(separator)
+        puts keywords
+      end
+
+      File.write(keywords_path, keywords)
+    end
+  end
```

Comment:
I got tired of manually tweaking the `sa-AR` keywords every submission, so I wrote this automation to do it.

Then, I realized that the best place to run this is after downloading the translations from GlotPress, so that the problem doesn't arise. That will require updating the Swift download script, and since I already invested time on this today, I decided to leave it for another time.

Reply:
@AliSoftware thanks for looking into this. You made realized I was working with the wrong assumption. 

When I wrote that code, I had in my mind that the keywords changed on every release. _I'm not sure why, as it doesn't make much sense in hindsight. Maybe because I got the error from running `deliver` which I associate to updating release-specific metadata, even though it does more than that, of course._ 🤔 🤷‍♂️ 

If they were ephemeral, so to speak, trimming them seems to me like a reasonable compromise to get through the release process smoothly. 

But ☝️, that's not the case for the keywords. In such case it makes total sense to fix the problem at the root. _It_ always _makes sense to fix the problem at the root, of course, but the fact that the keywords don't change that often means that trimming them every time on the client is indeed "not the right solution for the problem"._

I'm going to merge this PR and follow up with one that removes the lane 👍